requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+git://github.com/openstack/fuel-devops.git@2.9.13
+git+git://github.com/openstack/fuel-devops.git@2.9.14
```

Comment:
I believe we don't need to do it. Otherwise, our tests would depend from deployment scripts and couldn't use separately. Also we may have some problem with fuel-devops in tests (e.g. updating from 2.9.12 to 2.9.13)